client/src/main/java/com/tencent/rss/client/impl/ShuffleReadClientImpl.java

```diff
@@ -94,16 +93,20 @@ public ShuffleReadClientImpl(
     request.setHadoopConf(hadoopConf);
     request.setExpectBlockIds(blockIdBitmap);
     request.setProcessBlockIds(processedBlockIds);
+    request.setIndexLength(blockIdBitmap.getLongCardinality());
+
     List<Long> removeBlockIds = Lists.newArrayList();
     blockIdBitmap.forEach(bid -> {
-        if (!taskIdBitmap.contains(bid & Constants.MAX_TASK_ATTEMPT_ID)) {
-          removeBlockIds.add(bid);
+          if (!taskIdBitmap.contains(bid & Constants.MAX_TASK_ATTEMPT_ID)) {
```

Comment:
indent correct？

Reply:
Should be. My idea's auto format don't work?